static/js/containers/FinancialAidCalculator.js

```diff
@@ -18,26 +20,35 @@ import {
   setConfirmSkipDialogVisibility,
 } from '../actions/ui';
 import { createSimpleActionHelpers } from '../lib/redux';
-import SelectField from '../components/inputs/SelectField';
 import { currencyOptions } from '../lib/currency';
 import { validateFinancialAid } from '../lib/validation/profile';
 import { sanitizeNumberString } from '../lib/validation/date';
 import type { AvailableProgram } from '../flow/enrollmentTypes';
 import type {
   FinancialAidState,
   FinancialAidValidation,
+  FetchError,
 } from '../reducers/financial_aid';
 import type { Program } from '../flow/programTypes';
 import { formatPrice } from '../util/util';
 
+const updateCurrency = R.curry((update, financialAid, selection) => {
+  let _financialAid = R.clone(financialAid);
+  _financialAid.currency = selection ? selection.value : null;
+  update(_financialAid);
+});
+
 const currencySelect = (update, current) => (
-  <SelectField
+  <Select
     options={currencyOptions}
-    keySet={['currency']}
-    profile={current}
-    updateProfile={update}
+    value={current.currency}
+    onChange={updateCurrency(update, current)}
     name="currency"
     id="currency-select"
+    inputProps={{
+      'aria-required': "true",
```

Comment:
I believe the [`required`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-required) attribute should be used rather than `aria-required`. Most assistive tech should treat it the same, but with `required` you also get to use [the browser's built-in JavaScript form validation APIs](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/Data_form_validation#Validating_forms_using_JavaScript), if you want.